apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx

```diff
@@ -971,25 +1005,36 @@ const ImageLoadingGeneric = withV3StyleProvider(ImageLoadingGenericWithoutStyleP
 export const renderImageLoadRequested = ({
   t,
   device,
+  restore,
   type,
 }: {
   t: TFunction;
   device: Device;
+  restore: boolean;
   type: Theme["theme"];
 }) => {
   return (
     <ImageLoadingGeneric
-      title={t("customImage.steps.transfer.allowPreview", {
-        productName: device.deviceName || getDeviceModel(device.modelId)?.productName,
-      })}
+      restore={restore}
+      title={t(
+        restore
+          ? "customImage.steps.transfer.allowConfirmPreview"
+          : "customImage.steps.transfer.allowPreview",
+        {
+          productName: device.deviceName || getDeviceModel(device.modelId)?.productName,
+        },
+      )}
       progress={0}
       testId="device-action-image-load-requested"
     >
-      <FramedImage
-        background={
-          <Animation animation={getDeviceAnimation(device.modelId, type, "allowManager", true)} />
-        }
-      />
+      <DeviceBlocker />
+      <AnimationWrapper modelId={device.modelId}>
```

Comment:
`AnimationWrapper` is just a `div` no ? I saw that there were also a `modelId` prop on other usage of this `AnimationWrapper` in this file, but it just does nothing no ?